### PR TITLE
Release/1.9.0

### PR DESCRIPTION
**Pull request by vnarah2**
This branch has two features:
#32 #68 

A few things to remember when deploying this:
- Create UserAllocations, GroupAllocations, UserGroups collections. Create UserFinalQuota View.
-  Run the script in dev-scripts repo to update the usage information and className in some collections
- Update SV units in `commondb.Demand` to accept `in/s`
- Update the EQ threshold values in the code if needed - they are all set to 0.01 for now

### CODE REVIEW

File: server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/utils/JsonUtils.java [hunk 1]
```diff
@@ -49,4 +60,150 @@ public static String extractValueFromJsonString(String inId, String inJson) {
...
+
+        return userName;
+    }
+
+    public static Boolean isLoggedInUserAdmin(String userGroup){
+        Boolean isAdmin = false;
+
+        try {
+            JSONParser parser = new JSONParser();
+            org.json.simple.JSONObject userGroupJson = (org.json.simple.JSONObject) parser.parse(userGroup);
```
Comment (vnarah2) [inline server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/utils/JsonUtils.java:199]: This method throws an exception and returns 500 response code when the header doesn't contain `x-auth-usergroup` header (userGroup arg is null). It should fail gracefully with an appropriate API response message

File: server/space-service/src/main/java/edu/illinois/ncsa/incore/service/space/controllers/AllocationsController.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            throw new IncoreHTTPException(Response.Status.FORBIDDEN, "Logged in user is not incore admin");
+        }
+
+        return outJson.toString();
+    }
+
+    @GET
+    @Path("groups/{groupname}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Gives the allocation status of the given username.",
```
Comment (vnarah2) [inline server/space-service/src/main/java/edu/illinois/ncsa/incore/service/space/controllers/AllocationsController.java:109]: should be: Gives the allocation status of the given group name

File: server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/utils/JsonUtils.java (hunk 1 above)
Comment (ywkim312) [reply server/incore-common/src/main/java/edu/illinois/ncsa/incore/common/utils/JsonUtils.java:199]: updated

File: server/space-service/src/main/java/edu/illinois/ncsa/incore/service/space/controllers/AllocationsController.java (hunk 2 above)
Comment (ywkim312) [reply server/space-service/src/main/java/edu/illinois/ncsa/incore/service/space/controllers/AllocationsController.java:109]: changed

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**